Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -169,7 +169,8 @@ initData = {
     , openChatScreen : false
     , emergencyHelpModelState : emergencyHelpModalData
     , showLiveDashboard : false
-    , isBanner : true
+    , isProfileBanner : false
+    , isUpdateBanner : false
```

Comment:
@prakharritik instead of maintaining two different variables for banners can we create an enum for this, which would store the type of banner.

Reply:
RESOLVED.